### ISSUE TITLE
uninitialized constant Net::HTTP

### ISSUE THREAD

**Issue by PragmaticEd** [hisea/devise-bootstrap-views#52]
ruby version: `ruby 2.3.0p0 (2015-12-25 revision 53290) [x86_64-linux]`
os version: `ubuntu  lucid 32bit inside vagrant`
rails version: `5.0.0`
devise version: `4.2.0`

`rails g devise:views:locale en`
```
/home/vagrant/.rvm/gems/ruby-2.3.0/bundler/gems/devise-bootstrap-views-4175430cd32c/lib/generators/devise/views/locale/locale_generator.rb:7:in `download_locale': uninitialized constant Net::HTTP (NameError)
        from /home/vagrant/.rvm/gems/ruby-2.3.0/gems/thor-0.19.4/lib/thor/command.rb:27:in `run'
        from /home/vagrant/.rvm/gems/ruby-2.3.0/gems/thor-0.19.4/lib/thor/invocation.rb:126:in `invoke_command'
        from /home/vagrant/.rvm/gems/ruby-2.3.0/gems/thor-0.19.4/lib/thor/invocation.rb:133:in `block in invoke_all'
        from /home/vagrant/.rvm/gems/ruby-2.3.0/gems/thor-0.19.4/lib/thor/invocation.rb:133:in `each'
        from /home/vagrant/.rvm/gems/ruby-2.3.0/gems/thor-0.19.4/lib/thor/invocation.rb:133:in `map'
        from /home/vagrant/.rvm/gems/ruby-2.3.0/gems/thor-0.19.4/lib/thor/invocation.rb:133:in `invoke_all'
        from /home/vagrant/.rvm/gems/ruby-2.3.0/gems/thor-0.19.4/lib/thor/group.rb:232:in `dispatch'
        from /home/vagrant/.rvm/gems/ruby-2.3.0/gems/thor-0.19.4/lib/thor/base.rb:444:in `start'
        from /home/vagrant/.rvm/gems/ruby-2.3.0/gems/railties-5.0.2/lib/rails/generators.rb:180:in `invoke'
        from /home/vagrant/.rvm/gems/ruby-2.3.0/gems/railties-5.0.2/lib/rails/commands/generate.rb:13:in `<top (required)>'
        from /home/vagrant/.rvm/gems/ruby-2.3.0/gems/railties-5.0.2/lib/rails/commands/commands_tasks.rb:138:in `require_command!'
        from /home/vagrant/.rvm/gems/ruby-2.3.0/gems/railties-5.0.2/lib/rails/commands/commands_tasks.rb:145:in `generate_or_destroy'
        from /home/vagrant/.rvm/gems/ruby-2.3.0/gems/railties-5.0.2/lib/rails/commands/commands_tasks.rb:60:in `generate'
        from /home/vagrant/.rvm/gems/ruby-2.3.0/gems/railties-5.0.2/lib/rails/commands/commands_tasks.rb:49:in `run_command!'
        from /home/vagrant/.rvm/gems/ruby-2.3.0/gems/railties-5.0.2/lib/rails/commands.rb:18:in `<top (required)>'
        from bin/rails:4:in `require'
        from bin/rails:4:in `<main>'
```

**Comment by mlsaito** [comment]
The solution has been merged already right? I still got the error even though I just added this gem to my Gemfile and bundled it a few minutes ago. I was able to fix it by manually adding the fix to my locale_generator.rb though.

**Comment by hisea** [comment]
@mlsaito can you please try now? I just released version 0.11.

**Comment by mlsaito** [comment]
@hisea Just replicated the steps in a new project and it's working perfectly now. Thank you very much.
